docs/how-to/tls.rst

```diff
@@ -137,8 +137,104 @@ plaintext to TLS:
 * run ``microovn certificates regenerate-ca`` on one of the cluster members
 * run ``sudo snap restart microovn.daemon`` on **all** cluster members
 
-Once this is done, OVN services throughout the cluster will start listening on
-TLS-secured ports.
+Once this is done, OVN API services throughout the cluster will start listening
+on TLS-secured ports. However, the process is not complete yet because OVN
+Southbound and Northbound database clusters themselves are not capable of
+automatically switching to TLS communication in existing clusters. Both
+database clusters need to be manually switched over by individually removing
+cluster members that use ``tcp`` connection and reconnecting them with ``ssl``.
+This process technically replaces every member in the original cluster, but
+because we are doing it gradually, cluster data remains intact.
+
+Let's assume that we have a 3 node cluster. We'll start with switching over
+the ``OVN Northbound`` cluster.
+
+**Preparation**: We will be running commands on multiple nodes throughout this
+process, it is recommended to open a separate shell on each node and keep it
+open with following variables exported:
+
+.. code-block:: none
+    CONTROL_SOCKET=/var/snap/microovn/common/run/ovn/ovnnb_db.ctl
```

Comment:
Please align text directly under the **c** in `code-block`.

Here, and everywhere else.